### DISCUSSION TITLE
The future of the visual editor

### DISCUSSION THREAD

**Discussion by benifest** [korlibs/korge#2210] [Q&A]
Hey guys. The KTree editor https://korge.org/features/editor is marked as deprecated. I would like to ask is there a plan to have an ui editor for the Korge engine?

**Comment by rafi0** [comment]
Hi! I'm not speaking on behalf of the creator, but as far as I know, the Ktree is hard to maintain - hence the deprecation.
The go-to solution is to write code and use the hot-reloading mechanism. Personally I find this solution acceptable.

**Comment by soywiz** [comment]
Yes. There are plans. But likely not with KTree as it is right now. For now as rafi0 said, you can use hot reloading.
Eventually there will be something, but no ETA and no specific details on how.
